shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/Address/Bootstrap.hs

```diff
@@ -176,19 +171,21 @@ unpackByronVKey
     -- is the correct one. (32 bytes). If the XPub was constructed correctly,
     -- we already know that it has this length.
     Nothing -> panic "unpackByronVKey: impossible!"
-    Just vk -> (VKey vk, ChainCode chainCodeBytes)
+    Just vk -> (BootstrapVerKey vk, ChainCode chainCodeBytes)
 
 verifyBootstrapWit ::
   forall crypto.
-  (Crypto crypto, DSIGN.Signable (DSIGN crypto) (Hash crypto (TxBody crypto))) =>
+  (Crypto crypto, DSIGN.Signable DSIGN.Ed25519DSIGN (Hash crypto (TxBody crypto))) =>
   Hash crypto (TxBody crypto) ->
   BootstrapWitness crypto ->
   Bool
 verifyBootstrapWit txbodyHash witness =
-  verifySignedDSIGN
-    (bwKey witness)
-    txbodyHash
-    (bwSig witness)
+  either (const False) (const True) $
```

Comment:
OK to circumvent `verifySignedDSIGN` here?

Reply:
Yeah.